kratos/python/add_containers_to_python.cpp

```diff
@@ -81,6 +81,14 @@ template< class TBinderType, typename TContainerType, typename TVariableType > v
     binder.def("Clear", [](TContainerType& container){container.Clear();} );
 }
 
+template< class TBinderType, typename TContainerType, typename TVariableType > void DataValueContainerIndexingUtility(TBinderType& binder)
+{
+    // Data value container
+    VariableIndexingUtility<TBinderType, TContainerType, TVariableType>(binder);
+    binder.def("remove", [](TContainerType& container, const TVariableType& rV){return container.Erase(rV);} );
```

Comment:
You (@maceligueta ) ask me to add something pythonic

Reply:
You (@maceligueta ) ask me to add something pythonic

> @KratosMultiphysics/technical-committee agrees with adding only the `Erase` method and not the `del` because the second one implies deleting something in python and not cpp.
> 
> Nevertheless, we think that `remove` method is redundant

Removed

> @KratosMultiphysics/technical-committee agrees with adding only the `Erase` method and not the `del` because the second one implies deleting something in python and not cpp.
> 
> Nevertheless, we think that `remove` method is redundant



> @KratosMultiphysics/technical-committee agrees with adding only the `Erase` method and not the `del` because the second one implies deleting something in python and not cpp.
> 
> Nevertheless, we think that `remove` method is redundant



> @KratosMultiphysics/technical-committee agrees with adding only the `Erase` method and not the `del` because the second one implies deleting something in python and not cpp.
> 
> Nevertheless, we think that `remove` method is redundant

Done